utils/conversion/conversion_wrapper.go

```diff
@@ -0,0 +1,49 @@
+package conversion
+
+import (
+	"fmt"
+	"time"
+
+	specqbft "github.com/ssvlabs/ssv-spec/qbft"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+
+	"github.com/ssvlabs/ssv/exporter/convert"
+)
+
+var (
+	ErrNegativeTime = fmt.Errorf("time can't be negative")
+)
+
+// LenUint64 - Returns the length of the slice as uint64, due length can't be negative
+func LenUint64[T any](slice []T) uint64 {
+	return uint64(len(slice))
+}
+
+func TimeDurationFromUint64(t uint64) time.Duration {
+	return time.Duration(t) // #nosec G115
+}
+
+func BeaconRoleToConvertRole(beaconRole spectypes.BeaconRole) convert.RunnerRole {
+	return convert.RunnerRole(beaconRole) // #nosec G115
+}
+
+func BeaconRoleToRunnerRole(beaconRole spectypes.BeaconRole) spectypes.RunnerRole {
+	return spectypes.RunnerRole(beaconRole) // #nosec G115
+}
+
+func RunnerRoleToBeaconRole(role spectypes.RunnerRole) spectypes.BeaconRole {
+	return spectypes.BeaconRole(role) // #nosec G115
+}
+
+// TODO fix the type in spec and remove this func
+func CutoffRoundUint64() uint64 {
+	return uint64(specqbft.CutoffRound) // #nosec G115
+}
```

Comment:
if it's a constant in the spec's `qbft` package, can we just duplicate it in our respective `qbft` package instead of declaring a special func?

maybe `(var|const) CutoffRound = specqbft.CutoffRound` would work there